src/Lykke.Service.CandlesHistory.Services/Candles/RedisCachedCandle.cs

```diff
@@ -19,8 +19,6 @@ public class RedisCachedCandle
         [Key(6)]
         public double TradingOppositVolume { get; set; }
         [Key(7)]
```

Comment:
Is it safe to delete a field?

Reply:
Actually, Redis cache is cleaned up and filled with new data at every Lykke.Job.CandlesHistoryWriter restart. We are going to update and restart the *Writer and the reader service of this cache simultaneously, so, they will have similar awaited structure of the cache content.